src/Bicep.Cli/Services/Validation.cs

```diff
@@ -1,33 +1,49 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.IO;
 using System.Text;
 using Bicep.Core.Emit;
 using Bicep.Core.Intermediate;
 using Bicep.Core.Semantics;
 using Bicep.Core.Syntax;
+using Microsoft.Extensions.Logging;
 using Microsoft.WindowsAzure.ResourceStack.Common.Collections;
 using Microsoft.WindowsAzure.ResourceStack.Common.Json;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 
 namespace Bicep.Cli.Services
 {
-    public class ValidationService
+    public class Validation
```

Comment:
After looking through the other files, what about the following for naming:
* `TestRunner` -> produces a `TestRun` (or `TestResults`)
* `TemplateEvaluator` -> produces a `TemplateEvaluation`